nncf/experimental/onnx/datasets/segmentation_dataset.py

```diff
@@ -16,12 +16,12 @@
 from nncf.common.utils.logger import logger as nncf_logger
 from nncf.experimental.onnx.tensor import ONNXNNCFTensor
 
-from nncf.experimental.post_training.api.dataset import Dataset
+from nncf.quantization.dataset import PTQDataset
```

Comment:
I didn't find any entity with the name `PTQDataset`. @KodiaqQ, can you check this?

Reply:
Removed this.